tests/test_chebyshev.py

```diff
@@ -97,7 +97,7 @@ def test_chebyshev_evaluate(self, seed=1000):
         field = afwMath.ChebyshevBoundedField(bbox, self.pars)
         fieldValues = field.evaluate(xPos, yPos)
 
-        self.assertFloatsAlmostEqual(fieldValues, fgcmValues, rtol=5e-15)
+        self.assertFloatsAlmostEqual(fieldValues, fgcmValues, rtol=1e-14)
```

Comment:
Is three uses enough for this to be a file constant?

Reply:
They only need to be the same if we think a change in one tolerance should be reflected in the others. The number of uses is irrelevant in and of itself if the reused constant merely happens to be the same.